silo/gviews_v4.py

```diff
@@ -215,10 +215,7 @@ def import_from_gsheet_helper(user, silo_id, silo_name, spreadsheet_id, sheet_id
             except IndexError as e:
                 #this happens when a column header is missing gsheet
                 continue
-            if key == "" or key is None or key == "silo_id": continue
-            elif key == "id" or key == "_id": key = "user_assigned_id"
-            elif key == "edit_date": key = "editted_date"
-            elif key == "create_date": key = "created_date"
+            key = cleanKey(key)
```

Comment:
How can you prove that these changes are not breaking anything? This means we should have a unit test